src/fs/copy.js

```diff
@@ -1,3 +1,21 @@
+import * as fs from 'fs/promises';
+
 export const copy = async () => {
-    // Write your code here 
-};
\ No newline at end of file
+  fs.realpath('src/fs/files/')
+    .then(
+      res => {
+        fs.realpath('src/fs/files_copy/')
+          .then(() => {throw 'FS operation failed'})
```

Comment:
Task: if files folder doesn't exists or files_copy has already been created Error with message FS operation failed must be thrown
Message should be wrapped in Error()